### PR TITLE
Add files via upload

### LINKED ISSUE #36: UI Issue - Fix Email with signup Button in Home Page
### 🐞 Describe the Bug

* Fix `Email with signup` button on the Home page (hero and footer section)

image

### 💡 Expected behaviour

It should be like the Figma design.

### 📹 Screenshots or Video or GIF

Here's the Figma link,  
https://www.figma.com/file/nUaDVjPxi4qn0CTZNoLj0E/Feminine-Divine-Woodog?node-id=0%3A1
(Refer Home, Home-Mobile Frames)

---

**Note:**
* Do mention if you're working on this issue under ` LGM SOC 2021 `.
* If you want to work on any issue, you should check if it has already been assigned to anyone.

### PR DESCRIPTION
**Pull request by Sejal2801**
## Related Issue

Fixes #36

## Checklist:

- [ ] Have you checked there aren't other open [Pull Requests](https://github.com/Feminine-Divine/Woodog/pulls) for the same update/change?
- [ ] Your submission doesn't break any existing feature.
- [ ] Have you tested the code before submission?

## Screenshots or Video or GIF:

---

### CODE REVIEW

**Comment by khushishikhu** [comment]
@Sejal2801 add the issue number in your PR

**Comment by Sejal2801** [comment]
Okay

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-    "python.pythonPath": "env\\Scripts\\python.exe"
+    "python.pythonPath": "C:\\Users\\sejal\\AppData\\Local\\Programs\\Python\\Python36\\python.exe"
```
Comment (hariramjp777) [inline .vscode/settings.json:2]: Hi @Sejal2801, Please Don't change this line in `.vscode/settings.json`
Please look into <a href="https://github.com/Feminine-Divine/Woodog/blob/master/README.md">`readme.md`</a> and there you'll get instructions on how to use a virtual environment.

**Review by hariramjp777 — COMMENTED** [review]
@Sejal2801, Can you please make a new PR?

**Comment by Sejal2801** [comment]
Sir it has sent by mistake . My issue was #36 i have completed please see
that

**Comment by hariramjp777** [comment]
@Sejal2801 Yes, You've completed it. That's good :+1: But you have got a lot of changed files in your PR (like `woodog-master/`, `settings.json`).
That's why Please make a new PR.

If you're comfortable updating the same branch, you're welcome to do that.

**Comment by Sejal2801** [comment]
Sir i have not

**Comment by Sejal2801** [comment]
Sir i have not done any changes but why are these showing i dont know

**Comment by khushishikhu** [comment]
@Sejal2801 I will test your PR and then let you know where is the problem

**Comment by Sejal2801** [comment]
Okay

**Comment by khushishikhu** [comment]
I have tested this PR and can't find any issues. so merging this

**Review by khushishikhu — APPROVED** [review]

**Comment by Sejal2801** [comment]
Mam it was sent by mistake.sorry for that .my pc is not working properly.

**Comment by khushishikhu** [comment]
else I don't know why there are changes in settings.py and Readme and code of conduct. Although the purpose of the issue is completed. And no need to say mam :)

**Comment by Sejal2801** [comment]
Okay 😅

File: Woodog/templates/woodogdata/index.html [hunk 2]
```diff
@@ -79,15 +79,15 @@
...
+                <input type="email"  class="email-input form-control col-md-8" id="inputEmail4" placeholder="Email">
                 <button class="btn col apply-btn" type="submit">Sign up</button>
             </div>
         </form>
     </div>  
 </main>
 
 <div class="cards-section">
-    <p class="main-heading">How can you help?</p>
+    <p class="main-heading">How can you elp?</p>
```
Comment (Aaishpra) [inline Woodog/templates/woodogdata/index.html:90]: This PR needs to get reverted

File: Woodog-master/README.md [hunk 3]
```diff
@@ -0,0 +1,306 @@
+<p align = 'center'>
+<img src = "https://img.shields.io/badge/Woodog-4285F4?style=%2for-the-badge&logo=datadog&logoColor=white" height = '100' width = '300'/>
+</p>
+
+Woodog:dog: is an initiative to provide help to increasing stray animals by providing them food :bread: and refuge :house: by connecting them with fellow helpful humans.
+On the application, two types of people can be registered :inbox_tray: one who can provide the location and condition of a nearby stray, and another who can provide information and arrange shelter :house: and food :bread: for them.
```
Comment (Aaishpra) [inline Woodog-master/README.md:6]: This is all wrong another folder has been created which is not required at all

**Review by Aaishpra — COMMENTED** [review]
Reverting this PR

**Pull request merged**